packages/storybook/src/migrations/update-14-1-8/change-storybook-targets-generator.spec.ts

```diff
@@ -8,7 +8,9 @@ import * as extraOptions from './test-configs/extra-options-for-storybook.json';
 import * as noStorybookBuildTarget from './test-configs/no-build-storybook-target.json';
 import * as noStorybook from './test-configs/no-storybook-targets.json';
 
-describe('Change the Storybook targets for Angular projects to use native Storybooke executor', () => {
+// @TODO (@mandarini) Fixup and re-enable these tests. The snapshots are all the same currently,
+// and likely need to check **project** config, not nx.json config.
+xdescribe('Change the Storybook targets for Angular projects to use native Storybooke executor', () => {
```

Comment:
An alternative to the TODO here would just be to delete the migrations for 14.1.8.

They're less likely to be needed in Nx 17.

If a user is on a version < 14.1.8, they should be migrating to Nx 15 first anyway, which will contain the migrations.

Reply:
Yeah, that should probably happen but will be outside this PR 😄

Reply:
Yep, I will remove that migration!

Reply:
Oh! https://github.com/nrwl/nx/pull/19552 Jason already did!